subsys/mpsl/Kconfig.fem

```diff
@@ -72,4 +72,12 @@ config MPSL_FEM_NRF21540_RX_GAIN_DB
 
 endif
 
+config MPSL_FEM_DEVICE_CONFIG_254
+	bool "Apply device configuration 254"
+	default y
```

Comment:
Shouldn't it be `default MPSL_FEM`?

Reply:
It is enclosed with `if MPSL_FEM`, so there are no `MPSL_FEM_*` symbols if there is no FEM. Hence using `MPSL_FEM` yields the same result, however using plain `y` is more readable for me.
Please correct me if I am wrong, or using `MPSL_FEM` symbol is still more readable in your opinion.

Reply:
But this option should be available to users even when FEM is disabled. User should be able to decide to enable or disable config-254 regardless FEM is present in the system.

Reply:
@hubertmis, in that case, what should be the default value of the Kconfig option in case there is no FEM?

Reply:
Like @czeslawmakarski suggested: `default MPSL_FEM`, but overrideable by user. This default should work for most devices, but it depends on specifics of given device if this option shall be enabled.